demisto_sdk/commands/common/hook_validations/integration.py

```diff
@@ -498,9 +500,48 @@ def is_docker_image_valid(self):
 
     def is_valid_feed(self):
         # type: () -> bool
+        valid_from_version = valid_feed_params = True
         if self.current_file.get("script", {}).get("feed"):
             from_version = self.current_file.get("fromversion", "0.0.0")
             if not from_version or server_version_compare("5.5.0", from_version) == 1:
                 print_error(Errors.feed_wrong_from_version(self.file_path, from_version))
-                return False
-        return True
+                valid_from_version = False
+            valid_feed_params = self.is_valid_fetch()
+        return valid_from_version and valid_feed_params
+
+    def is_fetch_params_exist(self) -> bool:
```

Comment:
this one should be named is_valid_fetch()
because it checks if it fetches and than checks if the fields are valid.